modules/terraform/plan_struct_test.go

```diff
@@ -0,0 +1,78 @@
+package terraform
+
+import (
+	"testing"
+
+	http_helper "github.com/gruntwork-io/terratest/modules/http-helper"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	basicJsonUrl      = "https://raw.githubusercontent.com/hashicorp/terraform-json/v0.8.0/testdata/basic/plan.json"
+	deepModuleJsonUrl = "https://raw.githubusercontent.com/hashicorp/terraform-json/v0.8.0/testdata/deep_module/plan.json"
+
+	changesJsonUrl = "https://raw.githubusercontent.com/hashicorp/terraform-json/v0.8.0/testdata/has_changes/plan.json"
+)
+
+func TestPlannedValuesMapWithBasicJson(t *testing.T) {
+	t.Parallel()
+
+	// Retrieve test data from the terraform-json project.
```

Comment:
Any reason not to copy the JSON files locally to avoid an external dependency in this test?

Reply:
Just trying to be very conservative in how we use MPL code. I don't want to be in a situation where we have mixed licensing here, because technically, those json files will have to be MPL if they are checked in here.

Reply:
Ah, makes sense. Perhaps mention that in a comment?

Reply:
Done in 57eb06dd